### DISCUSSION TITLE
maximum call stack size exceeded

### DISCUSSION THREAD

**Discussion by liaodehui1** [solidjs/solid#894] [Q&A] [answered]
I'm working on a project that uses solidjs@1.0.7,  and recently received a "maximum call stack size exceeded " error. The error is located by sourceMap at the following screenshots. Has this problem been solved in the new version?
![image](https://user-images.githubusercontent.com/39728398/159450013-033a7970-a7f0-453f-8a67-15a76c1359d2.png)
![image](https://user-images.githubusercontent.com/39728398/159450274-5ca63e47-3cf4-4455-9ebc-2e46dadd01d9.png)

**Comment by atk** [comment, marked as answer]
Do you have a cyclic reference in your store?
